src/Hud/Menu/MenuPlugin.cs

```diff
@@ -201,12 +201,12 @@ private void CreateMenu()
             MenuItem xpRateMenu = AddChild(MenuRootButton, "Xph & area", settingsHub.XpRateSettings.Enable, "F10");
             MenuItem areaName = AddChild(xpRateMenu, "Only area name", settingsHub.XpRateSettings.OnlyAreaName);
             AddChild(areaName, "Show latency", settingsHub.XpRateSettings.ShowLatency);
-            AddChild(areaName, "Latency color", settingsHub.XpRateSettings.LatencyTextColor);
+            AddChild(areaName, "Show fps", settingsHub.XpRateSettings.ShowFps);
```

Comment:
@TehCheat do we need those 2? they are pointing to same color.
http://i.imgur.com/rzxz1hq.png